.changeset/six-avocados-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/coin-tezos": patch
```

Comment:
Small detail sorry for that but we now only use `minor` for standard bump and reserve the patch for hotfixes 
```suggestion
"@ledgerhq/coin-tezos": minor
```